evm/src/all_stark.rs

```diff
@@ -111,6 +111,8 @@ fn ctl_arithmetic<F: Field>() -> CrossTableLookup<F> {
         vec![
             cpu_stark::ctl_arithmetic_base_rows(),
             cpu_stark::ctl_arithmetic_shift_rows(),
+            cpu_stark::ctl_sys_exc_check_rows(),
+            cpu_stark::ctl_prover_input_check_rows(),
```

Comment:
Each of these cross-table lookups adds an extra column to the CPU table, so ideally we'd like to have as few of them as possible. Luckily, we can just use the first CTL (`cpu_stark::ctl_arithmetic_base_rows()`) for range checks.